### ISSUE TITLE
Accessibility fix: Add a lang attribute

### ISSUE THREAD

**Issue by ThomasFerro** [ThomasFerro/thomas-ferro#1]
Warning from the accessibility audit : the `html` tag is missing a `lang` attribute.

Set it to `fr` for now, since there is no multi locale support yet.
